app/homepage.py

```diff
@@ -358,7 +358,7 @@ def build_login_modal():
         id="login_modal",
 
         # Prevents the user from closing the modal by clicking next to it
-        backdrop='static',
+        # backdrop='static',
```

Comment:
Should we keep this or remove it ?

Reply:
Very nice catch, I commented this line for debugging but it should not stay as is indeed!